common/model-views.cpp

```diff
@@ -720,9 +763,23 @@ namespace rs2
                         else
                         {
                             float tmp_value = value;
-                            if (ImGui::SliderFloat(id.c_str(), &tmp_value,
-                                range.min, range.max, "%.4f"))
+                            float temp_value_displayed = tmp_value;
+                            float min_range_displayed = range.min;
+                            float max_range_displayed = range.max;
+
+                            // displaying in cm instead of meters for D405
+                            if (should_be_displayed_in_cm)
+                            {
+                                temp_value_displayed *= 100.f;
+                                min_range_displayed *= 100.f;
+                                max_range_displayed *= 100.f;
+                            }
+
+                            if (ImGui::SliderFloat(id.c_str(), &temp_value_displayed,
+                                min_range_displayed, max_range_displayed, "%.4f"))
```

Comment:
Adjust the umber of decimal digits to the option's step so for depth units it will show the complete number

Reply:
Done